extension/README.md.args.mjs

```diff
@@ -0,0 +1,275 @@
+export const skipQuickStart = true;
+
+export const extraContents = `# 🚩 Challenge 5: 📺 A State Channel Application
+
+![readme-5](https://github.com/scaffold-eth/se-2-challenges/assets/55535804/a6ea79fe-3fa1-4a3b-96d0-e5c0d2286356)
+
+🐌 The Ethereum blockchain has great decentralization & security properties. These properties come at a price: transaction throughput is low, and transactions can be expensive (search term: blockchain trilemma). This makes many traditional web applications infeasible on a blockchain... or does it?
+
+🍰 A number of approaches to scaling have been developed, collectively referred to as layer-2s (L2s). Among them is the concept of payment channels, state channels, and state channel networks. This tutorial walks through the creation of a simple state channel application, where users seeking a service **lock** collateral on-chain with a single transaction, **interact** with their service provider entirely off-chain, and **finalize** the interaction with a second on-chain transaction.
+
+🧑‍🤝‍🧑 State channels really excel as a scaling solution in cases where a fixed set of participants want to exchange value-for-service at high frequency. The canonical example is in file sharing or media streaming: the server exchanges chunks of a file in exchange for micropayments.
+
+🧙 In our case, the service provider is a \`Guru\` who provides off-the-cuff wisdom to each client \`Rube\` through a one-way chat box. Each character of text that is delivered is expected to be compensated with a payment of \`0.01 ETH\`.
+
+📖 Read more about state channels in the [Ethereum Docs.](https://ethereum.org/en/developers/docs/scaling/state-channels/)
+
+❗ [OpenZepplin's ECDSA Library](https://docs.openzeppelin.com/contracts/2.x/api/cryptography#ECDSA) provides an easy way to verify signed messages, but for this challenge we'll write the code ourselves.
+
+## 🏅 Main Quests
+
+- 🛣️ Build a \`packages/hardhat/contracts/Streamer.sol\` contract that collects **ETH** from numerous client addresses using a payable \`fundChannel()\` function and keeps track of \`balances\`.
+- 💵 Exchange paid services off-chain between the \`packages/hardhat/contracts/Streamer.sol\` contract owner (the **Guru**) and **rube** clients with funded channels. The **Guru** provides the service in exchange for signed vouchers which can later be redeemed on-chain.
+- ⏱ Create a Challenge mechanism with a timeout, so that **rubes** are protected from a **Guru** who goes offline while funds are locked on-chain (either by accident, or as a theft attempt).
+- ⁉ Consider some security / usability holes in the current design.
+
+> 💬 Meet other builders working on this challenge and get help in the [State Channel Telegram](https://t.me/+k0eUYngV2H0zYWUx)!
+
+---
+
+## Checkpoint 0: 📦 Environment 📚
+
+> in the same terminal, start your local network (a blockchain emulator in your computer):
```

Comment:
remove "In the same terminal" part

Reply:
Fixed!

Yes, it is the same as in some of the previous extensions I will fix it there too!

Reply:
Can you check if the update is pushed?

Reply:
Sorry, I don't know what happened. Pushed now.